### DISCUSSION TITLE
Syncthing integration

### DISCUSSION THREAD

**Discussion by DenysAshikhin** [photoprism/photoprism#960] [Q&A] [answered]
Hi all,

I am a complete noob at docker and syncthing- but from my research, it seems like the two together would fit my needs perfectly together. 

What I am trying to accomplish:
1) Sync pictures from multiple devices with syncthing onto a single home server
2) On the same server, run photoprism which would index and tag everything nicely and act as a unified gallery (preferably with subfolders for different members of family).

My question, how can I go about doing that? Photoprism runs on a win10 home with wsl2. I know that in the configuration (yaml) file I can specify folders and mounts, but I have no clue how that actually works. For example, if I have a folder on the win10 host under e:\general\pictures - how would I have photoprism be able to see that folder for indexing from inside its container?

Any guides or sample code would be greatly appreciated (and requested)!

Thanks in advance,
Spark

**Comment by lastzero** [comment, marked as answer]
Windows users may need to [disable](img/docker-disable-wsl2.jpg) the WSL 2 based engine in *Docker Settings > General* to mount drives other than `C:`. Please use this [docker-compose.yml](https://dl.photoprism.org/docker/windows/docker-compose.yml) example to get started and [increase](img/docker-resources-advanced.jpg) the Docker memory limit to 4 GB or more, as the default of 2 GB may reduce indexing performance or cause restarts.

You can mount one or many folders like this:

```
volumes:
      - "D:/Pictures:/photoprism/originals/Pictures"
      - "E:/Friends:/photoprism/originals/Friends"
```

Documentation: https://docs.photoprism.org/getting-started/docker-compose/

**Reply by lsantos10** [reply]
WSL2 automatically mount your local drives under /mnt/c/... or /mnt/d/... if the only reason for using hyper-v on windows is the mounting drivers, we should change this instruction as WSL2 is much faster than hyper-V

**Reply by jwinston100** [reply]
I'm sorry, but when I use WSL2, my local drives are _not_ mounted under /mnt/c/... or /mnt/d/..,, etc. I cannot use mounted drives at all, so I have to use hyper-v.

**Reply by lastzero** [reply]
What Windows edition and version do you have installed? My understanding is that Microsoft has fixed / improved this, but only based on the feedback we received. Don't have any detailed information.

**Comment by jwinston100** [comment]
It's Windows 11 Pro.
